server/base/src/main/java/org/apache/accumulo/server/manager/state/TabletManagementIterator.java

```diff
@@ -232,17 +233,34 @@ public Entry<Key,Value> next() {
     };
 
     final Set<ManagementAction> actions = new HashSet<>();
-    final TabletMetadata tm =
-        TabletMetadata.convertRow(kvIter, TabletManagement.CONFIGURED_COLUMNS, false, true);
-
     Exception error = null;
+    TabletMetadata tm = null;
+    KeyExtent extent = null;
     try {
-      LOG.trace("Evaluating extent: {}", tm);
-      computeTabletManagementActions(tm, actions);
-    } catch (Exception e) {
-      LOG.error("Error computing tablet management actions for extent: {}", tm.getExtent(), e);
+      tm = TabletMetadata.convertRow(kvIter, TabletManagement.CONFIGURED_COLUMNS, false, true);
+    } catch (RuntimeException e) {
+      LOG.error("Failed to convert tablet metadata at row: {}",
+          keys.get(0) == null ? "no key" : keys.get(0).getRow(), e);
```

Comment:
It does not hurt to check for keys[0] being nuill, however looking at RowEncodingIterator.prepKeys() it does not seem like it would ever pass null but that could change in future.

Was wondering if RowEncodingIterator.prepKeys() would ever pass an empty list.  Seems like it will not.